app/models/study.rb

```diff
@@ -1494,6 +1494,11 @@ def self.delete_queued_studies
   #
   ###
 
+  # get a quick count of all public studies that have compliant metadata
+  def self.public_compliant
+    self.where(public: true).select { |s| s.metadata_file&.use_metadata_convention? }.count
+  end
```

Comment:
This can go either way, but I'd lean against putting this method here.  I think it's better in summary_stats_utils, or a similar analytics module/class.  Study is long enough even with just the basic model functionality, it would be good to have analaytics elsewhere

Reply:
Oh it actually is, but I forgot to take this out after I refactored.  Will remove it now.